format/FormatHDF5EigerNearlyNexus.py

```diff
@@ -40,18 +42,18 @@ def find_entries(nx_file):
 def is_eiger_nearly_nexus_file(filename):
     """
     A hacky function to check if this is an EIGER-flavoured nexus file
-
     """
-    import h5py
-
     # Get the file handle
     handle = h5py.File(filename, "r")
 
     # Find the NXmx entries
     entry = find_entries(handle)
     if entry is not None:
         try:
-            return "Dectris Eiger" in entry["instrument"]["detector"]["description"][()]
+            return (
```

Comment:
would suggest checking for `"dectris eiger" in ...[()].lower()` in case DECTRIS Eiger etc.

Reply:
done